File: server/Core/Defaults.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ZeusBrowser.Server.Core
{
    public static class Defaults
    {
		public const string AntiforgeryHeaderName = "X-Zeus-Lightning";
		public const string AntiforgeryCookieName = "Zeus-Lightning";
		public const string Path = "/";
		public const GroupBy GroupBy = Core.GroupBy.FileSystemInfoType;
		public const OrderBy OrderBy = Core.OrderBy.None;
		public const int StreamWriterBufferSize = 512 * 1024;
		public const string InternalUriScheme = "zb";
		public const string InternetUriHost = "zeus-browser";
		public const int MediaInfoProcessTimeout = 30 * 1000;
	}
}

File: client/src/defaults/defaults.ts
import { InjectionToken } from '@angular/core';

import { GroupBy } from '../app/shared/enums/group-by.enum';
import { OrderBy } from '../app/shared/enums/order-by.enum';

export let DEFAULTS = new InjectionToken('defaults');

export interface IDefaults {
	antiforgeryHeaderName: string;
	antiforgeryCookieName: string;
	path: string;
	groupBy: GroupBy;
	orderBy: OrderBy;
}

export const Defaults: IDefaults = {
	antiforgeryHeaderName: 'X-Zeus-Lightning',
	antiforgeryCookieName: 'Zeus-Lightning',
	path: '/',
	groupBy: GroupBy.FileSystemInfoType,
	orderBy: OrderBy.None
};
